app/models/badge.rb

```diff
@@ -0,0 +1,18 @@
+class Badge < ActiveRecord::Base
+  include Comparable
+  has_many :users
+
+  def self.get (input)
+    if input.is_a?(String) || input.is_a?(Symbol)
+      Badge.find_by_name(input)
```

Comment:
Sure, I guess. I can change it in `role.rb| also, as that's where I copied it from.